### PR TITLE
touca 1.8.6 (new formula)

### PR DESCRIPTION
**Pull request by ghorbanzade**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/touca.rb [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    sha256 "b75f22b57749ea362db87c532b74bf51fc757ec60492037fc10ca7371b7299bf"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/c5/52/fe421fb7364aa738b3506a2d99e4f3a56e079c0a798e9f4fa5e14c60922f/urllib3-1.26.14.tar.gz"
+    sha256 "076907bf8fd355cde77728471316625a4d2f7e713c125f51953bb5b3eecf4f72"
+  end
+
+  def install
+    virtualenv_create(libexec, "python3")
```
Comment (chenrui333) [inline Formula/touca.rb:142]: no need to have that
```suggestion
```

File: Formula/touca.rb [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  end
+
+  test do
+    assert_match "v#{version}", shell_output("#{bin}/touca version")
+    assert_equal "", shell_output("#{bin}/touca profile set test-profile")
+    assert_equal "", shell_output("#{bin}/touca config set some-key=some-value")
+    assert_match "some-key", shell_output("#{bin}/touca config show")
+    assert_match "some-value", shell_output("#{bin}/touca config show")
+    assert_match "test-profile", shell_output("#{bin}/touca profile ls")
+    assert_equal "", shell_output("#{bin}/touca profile rm test-profile")
```
Comment (chenrui333) [inline Formula/touca.rb:153]: improve test a bit
```suggestion
    assert_match "v#{version}", shell_output("#{bin}/touca version")
    assert_empty shell_output("#{bin}/touca profile set test-profile")
    assert_empty shell_output("#{bin}/touca config set some-key=some-value")
    assert_match "some-key", shell_output("#{bin}/touca config show")
    assert_match "some-value", shell_output("#{bin}/touca config show")
    assert_match "test-profile", shell_output("#{bin}/touca profile ls")
    assert_empty shell_output("#{bin}/touca profile rm test-profile")
```

Comment (ghorbanzade) [reply Formula/touca.rb]: Thank you for the suggestion! Fixed. :)

File: Formula/touca.rb (hunk 1 above)
Comment (ghorbanzade) [reply Formula/touca.rb]: Oh, great. I didn't know that. Fixed.

**Comment by ghorbanzade** [comment]
Thank you for the review @chenrui333. I applied the suggested changes and updated the PR.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ghorbanzade, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**